runtime/src/lib.rs

```diff
@@ -910,7 +910,7 @@ impl pallet_sminer::Config for Runtime {
 	type ItemLimit = ConstU32<10_000>;
 	type MultipleFines = MultipleFines;
 	type DepositBufferPeriod = DepositBufferPeriod;
-	type CalculFailureFee = Sminer;
+	type CalculFailureFee = Sminer; // this dependency with itself is really necessary?
```

Comment:
You're right. I got rid of self dependence:
https://github.com/CESSProject/cess/pull/88/files#diff-0ec06ea58bd455f09ce6b3bb4c2c1c0d37bda51c1e1be2151c560c9c973959ecL913